DiscordBot/src/commands/djsCommand.ts

```diff
@@ -38,7 +38,23 @@ export default class DjsCommand extends BaseCommand {
             .then(res => res.json())
             .then(embed => {
                 if (embed && !embed.error) {
-                    commandData.message.channel.send({ embed });
+                    commandData.message.channel.send({ embed }).then(async m => {
+                        await (m as Message).react('🗑')
```

Comment:
Missing ;

Reply:
Um, sorry, but I think I might have missed your point, where is the missing semi-colon at exactly? 
```js
await (m as Message).react(...)
```
? Because, if so, then that's some form of IDE popping up those warnings. They do not have actual effect on the code itself, nor it's efficiency, but, sure. Just let me know if that's where you wanted me to put it, because I'm sincerely confused about that one.